db/DBHelper.py

```diff
@@ -0,0 +1,150 @@
+import postgresql as pg
+from os import listdir
+from db.date.ItemInfo import ItemInfo
+
+CHECKS_TABLE = "\"Checks\""
+USERS_TABLE = "\"Users\""
+ITEMS_TABLE = "\"Items\""
+CATEGORIES_TABLE = "\"Categories\""
+PATTERNS_TABLE = "\"Patterns\""
+
+USER = "postgres"
+PASS = ""
+HOST = "localhost"
+PORT = "5432"
+DB_NAME = "QrCodesDB"
+
+DB_PARAMS = 'pq://' + USER + ':' + PASS + '@' + HOST + ':' + PORT + '/' + DB_NAME
+
+SCRIPTS_PATH = "scripts/"
+SEQ_SCRIPTS_PATH = SCRIPTS_PATH + "seq/"
+TAB_SCRIPTS_PATH = SCRIPTS_PATH + "tables/"
+
+
+# TODO: methods for statistic
+# TODO: methods for waiting chacks
+
+
+class DBHelper:
+
+    def __init__(self, local=True, user="", password="", host="", port="", db_name=""):
+        try:
+            if local:
+                try:
+                    self.__create_new_db()
+                    self.db = self.__conect_to_local()
+                except pg.exceptions.ClientCannotConnectError:
+                    print("<INFO>: Attempt to create a new database:\n" + DB_PARAMS)
+                    self.db = self.__create_new_db()
+            else:
+                self.db = self.__connect(user, password, host, port, db_name)
+        except pg.exceptions.ClientCannotConnectError:
+            print("CONNECTION ERROR")
+
+    #
+    # User API
+    #
+    def users(self):
+        return self.__selectAllQuery(USERS_TABLE)
+
+    def addUser(self, login, password):
+        # TODO: login&pass checking
+        values = "('" + login + "','" + password + "')"
+        self.__insertQuery(USERS_TABLE, "(login,password)", values)
+
+    def userExist(self, login):
+        query = """SELECT CASE WHEN EXISTS (
+                                    SELECT *
+                                    FROM %s
+                                    WHERE login = '%s'
+                    )
+                    THEN True
+                    ELSE False END""" % (USERS_TABLE, login)
+        return self.db.query(query)[0][0]
+
+    def userId(self, login):
+        return self.__selectQuery("id", USERS_TABLE, "WHERE login = '%s'" % login)[0][0]
+
+    #
+    # Checks API
+    #
+    def checks(self):
+        return self.__selectAllQuery(CHECKS_TABLE)
+
+    def addCheck(self, specifier, shop, date, login):
```

Comment:
```suggestion
    def addCheck(self, specifier, shop, date, user_login):
```
небольшое уточнение, в слующих функция так же